analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/metrics/CalendarWeek.kt

```diff
@@ -19,7 +20,16 @@ internal data class CalendarWeek(private val week: Int, private val year: Int) :
         }
 
         fun numberOfWeeksBetween(a: CalendarWeek, b: CalendarWeek): Int {
-            return b.week + 52 * b.year - (a.week + 52 * a.year)
+            return ChronoUnit.WEEKS.between(getWeekDate(a.year, a.week), getWeekDate(b.year, b.week)).toInt()
+        }
+
+        private fun getWeekDate(year: Int, week: Int): OffsetDateTime? {
+            // returns the date of Monday based on the week and year
```

Comment:
[Optional] I like to have longer variable names like `getDateOfWeeksMonday` instead of comments, which tend to rotten 😇 Guess that would also meet @BridgeAR wish for a comment.